### PR TITLE
Removed unnecessary "calculate" button

### LINKED ISSUE #90: Remove "Calculate" button in Binary Operations section
The **Calculate** button in the Binary Operations should be removed as it serves no purpose. The calculations happen as soon as we change the inputs or operations.

**Steps to Reproduce**
* Go to https://learn.circuitverse.org/docs/binary2.html
* Click on numbers and operations to see result, calculate button is not required.

**Are you working on this issue?**
* Nope. For GCI 2019

### PR DESCRIPTION
**Pull request by CodeSarthak**
Closes #90 

GCI task link : https://codein.withgoogle.com/dashboard/task-instances/5511844463116288/

### CODE REVIEW

**Comment by tachyons** [comment]
Add GCI task link in the PR description

File: _includes/binary2.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
         <div class="opcol">AND</div>
         <div class="opcol">AND</div>
         <div class="opcol">AND</div>
-    </div><input type="button" value="Calculate" onClick="set_bits();" class="ok" style="margin-top: 22px"><br style="clear: left">
+    </div>
```
Comment (deepak2431) [inline _includes/binary2.html]: @CodeSarthak I think you have added an extra closing div tag. Please check and update your PR.

Comment (Shweta0002) [reply _includes/binary2.html]: You can share a screenshot after removing  button also.

**Comment by CodeSarthak** [comment]
Closing this PR as a new one has been made at : #103

**Pull request closed without merging**